packages/strapi/__tests__/search.test.e2e.js

```diff
@@ -156,7 +156,7 @@ describe('Search query', () => {
     test('search for "id"', async () => {
       const res = await rq({
         method: 'GET',
-        url: '/content-manager/explorer/application::bed.bed',
+        url: '/beds',
```

Comment:
Why did you change the url?
It tests the search bar in the CM

Reply:
Because it was making calls to a plugin controller while testing public APIs (from the strapi package).

Reply:
Ok, then I think we should duplicate the tests for the CM too. What do you think?